.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: python
 python:
-- '2.6'
 - '2.7'
-- '3.2'
```

Comment:
we are not - see tox.ini, environments are all there instead
so now testing is not different locally and on travis, just by running tox command